modules/ROOT/pages/tgw-about.adoc

```diff
@@ -0,0 +1,49 @@
+= Transit Gateway Attachments
```

Comment:
I'm not sure if it's appropriate to mention attachments in the main heading. They _do_ have to accept an attachment in AWS for each VPC they want to connect to their TGW, but we represent this in the UI as just part of the flow, rather than main object they're creating/managing. IMO just "Transit Gateways" would be fine here.

Reply:
@evinaku - can you weigh in? (This came from Vinay.)

Reply:
I'm going to go with Transit Gateways and Taylor's recommendation. I'll let Vinay correct in his review, if necessary.